cudatoolkit-dev/cuda-install.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash 
+
+TEMPDIR=/tmp/cuda 
```

Comment:
This should probably be in the `$CONDA_PREFIX` directory somewhere because we may not be able to write to `/tmp` in all cases.

Reply:
That is right. I will change it to `$CONDA_PREFIX`.